Makefile

```
@@ -0,0 +1,31 @@
+LOCAL_BIN:=$(CURDIR)/bin
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)